ts/lib/auth/jwtclient.ts

```diff
@@ -74,7 +75,8 @@ export default class JWT extends Auth2Client {
    * @param {string} opt_uri the URI being authorized.
    * @param {function} metadataCb
    */
-  public getRequestMetadata(opt_uri: string, metadataCb) {
+  public getRequestMetadata(
+      opt_uri: string, metadataCb: (err: Error, result?: any) => void) {
```

Comment:
I would have recommended `opt_uri` => `uri?`, but I don't think those are the actual semantics here. I think this is supposed to be a nullable string? not really optional...